Makefile.operator

```diff
@@ -0,0 +1,36 @@
+OPERATOR_DIR=../docker-images/operator
+LIB_DIR=$(OPERATOR_DIR)/lib
+SCRIPTS_DIR=$(OPERATOR_DIR)/$(PROJECT_NAME)-scripts
+RELEASE_VERSION=$(shell cat ../release.version)
+JAR=io.strimzi.$(PROJECT_NAME)-$(RELEASE_VERSION).jar
+
+docker_build: java_build
+
+all: docker_build docker_push
+clean: java_clean
+
+include ../Makefile.maven
+
+docker_build:
+	mkdir -p $(LIB_DIR)
+# Find dependent jars which are signed (contain META-INF/*.SF or META-INF/*RSA) and remove those from the jar
+	find . -name '*.jar' -exec sh -c 'unzip -l {} | grep -qE '"'"'META-INF/*.(SF|RSA)'"'" \; -print \
+	  | xargs -I % zip -d % 'META-INF/*.SF' 'META-INF/*.RSA'
+	cp -r target/lib/* $(LIB_DIR)/
+	mv target/$(PROJECT_NAME)-$(RELEASE_VERSION).jar $(LIB_DIR)/$(JAR)
+# Copy all the scripts
```

Comment:
Can you indent the comments with the code? I think it will be easier to read.

Reply:
If you indent the comments `make` treats them as the recipe and they get echoed in the `make` output which I personally don't find helpful; it tends to put people off putting comments (because they'll be spewed out in the make output which makes it less easy to read), which is ultimately counter productive. I'll do it if you really want...?